### ISSUE TITLE
user_opts

### ISSUE THREAD

**Issue by klooj** [glepnir/indent-guides.nvim#17]
At least one of the keys from the pairs below is not registering as a default key, so none of the user_opts are taking effect.  Is this a problem in my config or a minor bug?

<img width="425" alt="image" src="https://user-images.githubusercontent.com/57016271/109044530-813c7500-76a0-11eb-9f06-4ba251d29f6d.png">

**Comment by klooj** [comment]
is it possible that the space occupied by the indent guide or the guide itself is registering as a character?

**Comment by klooj** [comment]
I tried to fix it but I cannot figure it out. if it's helpful, here's what I found along the way:

1. commit #cbe28c6 - everything works (before new feature)
2. commit #4887936f - dashboard generates the following error:
<img width="778" alt="grab-mbook 2021-02-24 at 4 30 07 PM" src="https://user-images.githubusercontent.com/57016271/109069766-a8566f00-76bf-11eb-971e-1460bd84210d.png">
3. commit #90ad6896, #cbe2b7a - no errors, but the guides are now visible in dashboard:
<img width="509" alt="grab-mbook 2021-02-24 at 4 35 15 PM" src="https://user-images.githubusercontent.com/57016271/109069894-d3d95980-76bf-11eb-842e-69ac2b0adbeb.png">
4. commit #5e1cc461 - guides are visible + new error:
<img width="466" alt="grab-mbook 2021-02-24 at 4 40 55 PM" src="https://user-images.githubusercontent.com/57016271/109069979-f1a6be80-76bf-11eb-8cac-4ef3cac3964e.png">

Let me know if I can do something useful.
